homeassistant/components/zwave_js/api.py

```diff
@@ -437,6 +551,14 @@ async def websocket_ping_node(
         vol.Optional(INCLUSION_STRATEGY, default=InclusionStrategy.DEFAULT): vol.In(
             [strategy.value for strategy in InclusionStrategy]
         ),
+        vol.Optional(FORCE_SECURITY): bool,
+        vol.Exclusive(
+            PLANNED_PROVISIIONING_ENTRY, "options"
+        ): PLANNED_PROVISIONING_ENTRY_SCHEMA,
+        vol.Exclusive(
+            QR_PROVISIONING_INFORMATION, "options"
+        ): QR_PROVISIONING_INFORMATION_SCHEMA,
+        vol.Exclusive(QR_CODE_STRING, "options"): str,
```

Comment:
We can use `cv.has_at_least_one_key` to require one of these items:
https://github.com/home-assistant/core/blob/b6dea3c6cb291c7df9282eb46e617a25e7d69c80/homeassistant/helpers/config_validation.py#L122-L123